tests/system/providers/google/cloud/cloud_sql/example_cloud_sql.py

```diff
@@ -51,12 +51,14 @@
 PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
 DAG_ID = "cloudsql"
 
-INSTANCE_NAME = f"{DAG_ID}-{ENV_ID}-instance"
-DB_NAME = f"{DAG_ID}-{ENV_ID}-db"
+INSTANCE_NAME = f"{DAG_ID}-{ENV_ID}-instance".replace("_", "-")
```

Comment:
May I know why do we need these `replace("_", "-")`. It seems to be `{DAG_ID}-{ENV_ID}-instance` on the deferrable side as well

Reply:
Thre cannot be `_` in the instance name I believe (and I think the change to deferrable is coming as well :)

Reply:
Thank you for taking a look on this PR!
You are right, the name of the instance is valid only if consists of dashes and letters only, in this case. 
As this PR combines 2 system tests into one, the deferrable and sync mode are combined in one file, so created instance will be used in both cases (def and sync)